src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -547,6 +548,14 @@ public int getOffsetPeriodDuration() {
     return getDuration("openingTimeOffset");
   }
 
+  public int getGracePeriodDuration() {
+    return getDuration(GRACE_PERIOD_KEY);
+  }
+
+  public LoanPolicyPeriod getGracePeriodInterval() {
+    return getPeriod(GRACE_PERIOD_KEY);
+  }
```

Comment:
Rather than using these separately, could the grace period be represented as a `TimePeriod`?

Reply:
I figured that it would be more appropriate to represent it as `Period`. What do you think?

Reply:
That is fine, the two need combining at some point (which is outside of the scope of this work)__